src/cljs/rems/spa.cljs

```diff
@@ -332,11 +335,16 @@
 
 (defn page []
   (let [page-id @(rf/subscribe [:page])
-        grab-focus? @(rf/subscribe [::grab-focus?])]
+        grab-focus? @(rf/subscribe [::grab-focus?])
+        theme @(rf/subscribe [:theme])
+        lang @(rf/subscribe [:user-language])]
     (lazy-load-data!)
     [:div
      [nav/navigation-widget]
-     [logo]
+     (when (or (= page-id :home)
+               (and (not ((keyword (str "navbar-logo-name-" (name lang))) theme))
+                    (not (:navbar-logo-name theme))))
+       [nav/logo])
```

Comment:
This logic is still wrong. The regular logo should be there always when configured so. It should not depend on the page id or navbar logo.